package.json

```diff
@@ -36,26 +36,26 @@
   },
   "homepage": "https://github.com/Agoric/documentation#readme",
   "dependencies": {
-    "@agoric/zoe": "beta",
-    "@agoric/ertp": "beta",
+    "@agoric/zoe": "community-dev",
+    "@agoric/ertp": "community-dev",
     "@endo/far": "^0.2.19",
     "@endo/marshal": "^0.8.6",
     "@endo/patterns": "^0.2.3",
     "@endo/pass-style": "^0.1.6",
     "typescript": "^4.0.3"
   },
   "devDependencies": {
-    "@agoric/assert": "beta",
-    "@agoric/cosmic-swingset": "beta",
-    "@agoric/ertp": "beta",
-    "@agoric/notifier": "beta",
-    "@agoric/solo": "beta",
-    "@agoric/vats": "beta",
-    "@endo/bundle-source": "^2.1.1",
-    "@endo/eventual-send": "^0.14.8",
-    "@endo/init": "^0.5.43",
-    "@endo/promise-kit": "^0.2.37",
-    "@endo/ses-ava": "^0.2.27",
+    "@agoric/assert": "community-dev",
+    "@agoric/cosmic-swingset": "community-dev",
+    "@agoric/ertp": "community-dev",
+    "@agoric/notifier": "community-dev",
+    "@agoric/solo": "community-dev",
+    "@agoric/vats": "community-dev",
+    "@endo/bundle-source": "2.5.2-upstream-rollup",
+    "@endo/eventual-send": "^0.17.2",
+    "@endo/init": "^0.5.56",
+    "@endo/promise-kit": "^0.2.56",
+    "@endo/ses-ava": "^0.2.40",
```

Comment:
It's not obvious how these are related to community-dev.

Are they from package.json files from u13? Got a pointer?

Reply:
I grabbed the versions from `yarn.lock`, lines 1323 - 1523:

https://github.com/Agoric/agoric-sdk/blob/5a6cdeb0c18ae9700d706445acf402f8d1e873c3/yarn.lock#L1328

https://github.com/Agoric/agoric-sdk/blob/community-dev/yarn.lock#L1328